riak_test/tests/auth_bypass_test.erl

```diff
@@ -23,12 +23,14 @@
 -export([confirm/0]).
 -include_lib("eunit/include/eunit.hrl").
 -include_lib("erlcloud/include/erlcloud_aws.hrl").
+config() ->
+    [{riak_cs, [{admin_auth_enabled, false}]}].
 
 confirm() ->
-    Config = [{cs, rtcs:cs_config([{admin_auth_enabled, false}])}],
-    {UserConfig, {RiakNodes, _CSNodes, _Stanchion}} = rtcs:setup(1, Config),
+    rtcs_dev:set_advanced_conf(cs, config()),
```

Comment:
Because I don't understand their responsibility well, and many functions updating configs already have been there. It seems suitable to be in rtcs. Will do.